### DISCUSSION TITLE
Purchase GVRET CAN DUE

### DISCUSSION THREAD

**Discussion by Breastfed** [collin80/SavvyCAN#767] [General]
Hi, 
I'm looking to buy  the GVRET CAN Due board,  however there's no shipping options to Thailand.

BTW has the EVTV replaced the GVRET ?
Thank you.

Breastfed

**Comment by collin80** [comment]
I'm sorry about the lack of shipping options to Thailand. I don't have any control over that. You might try contacting the shop directly at sales@evtv.me to see if they can ship to you.

The GVRET board has been replaced by the ESP32 based board. This was partially because the SAM3X chips used for GVRET were discontinued for years. But, they have brought them back so we may be getting more boards made in the near future. The GVRET/EVTVDue board has the potential for two CAN buses and built-in USB. The ESP32 instead has an on-board USB to serial chip so it isn't as fast over USB but does support wifi and bluetooth so it can actually be faster via WiFi.

**Reply by DIYiphone** [reply]
Collin can you help me get in touch with someone from [EVTV due](https://store.evtv.me/products/evtvdue2)
